modules/monitoring-contents-of-the-alerting-ui.adoc

```diff
@@ -33,4 +33,17 @@ image::silences-screen.png[]
 . Number of alerts that are being silenced by the silence.
 . Actions you can do with a silence.
 
-Additionally, next to the title of each of these pages is a link to the old Alertmanager interface.
+The *YAML* page is located in *Monitoring* -> *Alerting* -> *YAML* of the OpenShift Container Platform web console.
+
+image::yaml-screen.png[]
+
+. Upload a file with Alertmanager configuration.
+. Examine and edit the current Alertmanager configuration.
+. Save the updated Alertmanager configuration.
+
+Also, next to the title of each of these pages is a link to the old Alertmanager interface.
```

Comment:
`..is a link to the old Alertmanager interface`   Not sure if 'old' is the best term here.   Maybe 'Promethus interface'?

Reply:
It's the term we agreed to use to imply that people should switch to the new OCP interface. And it's not the Prometheus interface anyway, the Prometheus one is linked from the Metrics page. This is the AM one.